project/Release.scala

```diff
@@ -29,7 +29,7 @@ object Release {
     val repo = extracted.get(Publish.defaultPublishTo)
     val state1 = extracted.runAggregated(publishSigned in projectRef, state)
     val (state2, Seq(api, japi)) = extracted.runTask(unidoc in Compile, state1)
-    val (state3, docs) = extracted.runTask(generate in Sphinx, state2)
+    val (state3, docs) = extracted.runTask(paradox in ProjectRef(projectRef.build, "akka-docs") in Compile, state2)
 
```

Comment:
Ah right - yes, this task now copies those to e.g. `target/release/docs/akka/2.5-SNAPSHOT/index.html` etc so that looks good